examples/simple-chat-client/src/bin/simple_chat.rs

```diff
@@ -0,0 +1,84 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use simple_chat_client::{
+    chat::ChatSession,
+    client::OpenAIClient,
+    config::Config,
+    tool::{Tool, ToolSet, get_mcp_tools},
+};
+
+//default config path
+const DEFAULT_CONFIG_PATH: &str = "/etc/simple-chat-client/config.toml";
```

Comment:
for an example app, should use `./config.toml` as to not need escalated privileges to read from `/etc`